aur/packages

```diff
@@ -2,6 +2,7 @@ abcm2ps
 abcpp
 aether.lv2
 a-sid
+audiowmark
```

Comment:
Since there is no `audiowmark` package in AUR yet, I'm not sure what will happen, when this is merged. Maybe @cbix can clarify.

Reply:
It will simply be added by the `osamc` user.